examples/allocator-service/README.md

```diff
@@ -1,10 +1,10 @@
 # Simple Allocator Service
 
-This service provides an example of using the [Agones API](https://godoc.org/agones.dev/agones/pkg/client/clientset/versioned/typed/stable/v1alpha1) to allocate a GameServer from a Fleet, and is used in the [Create an Allocator Service (Go)](../../docs/create_allocator_service.md) tutorial.
+This service provides an example of using the [Agones API](https://godoc.org/agones.dev/agones/pkg/client/clientset/versioned/typed/stable/v1alpha1) to allocate a GameServer from a Fleet, and is used in the [Create an Allocator Service (Go)](https://agones.dev/site/docs/tutorials/allocator-service-go/) tutorial.
 
 ## Allocator Service
```

Comment:
Should we get rid of this sample now that we have the proper game server allocator in place?

Reply:
Sounds like a plan for when the allocator is fully operational? It might be a bit premature now, WDYT?

(Insert Death Star references here :smile:)